datalad/distribution/add.py

```diff
@@ -119,8 +119,16 @@ class Add(Interface):
             transport"""),
         recursive=recursion_flag,
         recursion_limit=recursion_limit,
+        ds2super=Parameter(
+            args=("--ds2super", "--datasets-to-super",),
```

Comment:
But since it is impossible, imho it is OK not to disambiguate - doc would be enough imho.
Isee this option to be frequently used but ds2 prefix is ugly imho